IFC4x3/Properties/t/Transmissivity_1md2GJC9f2MflqkMhIDr4K/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Transmissivity_1md2GJC9f2MflqkMhIDr4K" Name="Transmissivity" UniqueId="709c2413-309a-425a-9bf4-b96ad2375114" PrimaryDataType="IfcReal" />
```

Comment:
Data type is problematic.

Reply:
Reused Transmissivity from Pset_InSituTestPumping.